src/constants.js

```diff
@@ -633,14 +633,37 @@ export const VIRAL_LOAD_ALERTS = [
 ];
 
 export const VIRAL_LOAD_ALERTS_CATEGORIES = [
```

Comment:
@rhenshaw56  you followed the pattern that already existed, which made sense, but taking a closer look I think we should change all these arrays into single category constants, ie:

`VIRAL_LOAD_ALERTS_CATEGORY = "viral-load"`

and then add a SCREENING_ELIGIBILITY_CATEGORY.  (There may be a time when we want to, for example, search for an alert that *just* has the blood pressure category).

Then all the filters would be updated to something like this:

`return utils.hasAlert(patient.alert, [ADHERENCE_ALERTS_CATEGORY, SCREENING_ELIGIBILITY_CATEGORY])`

if you wanted we could make a single const like:

```
const ALERT_CATEGORIES = {
    VIRAL_LOAD: "vl",
    HTC: "HTC"
  ...
}
```

Reply:
@mogoodrich I have made the changes the you requested

Reply:
Thanks @rhenshaw56 and @mogoodrich , this looks good!